bin/maxmind-geolite-mirror

```diff
@@ -1,30 +1,25 @@
 #!/usr/bin/env node
-
 const fs    = require('fs');
 const http  = require('http');
 const zlib  = require('zlib');
-
+const path = require('path');
```

Comment:
please restore the blank line here and include path in the top list of "built-in" modules. `config` is separated by whitespace because it's a local dependency.